### PR TITLE
Feat/release

### PR DESCRIPTION
**Pull request by meyerhp**
### Description
This PR updates the build scripts to create feature releases. Any branch with the `feat/` prefix will trigger a new release which will tag the code and create a pre-release on GitHub.

E.g release for this branch: https://github.com/SPANDigital/presidium-js/releases/tag/v2.13.0-feat-release.27

### Issue
https://spandigital.atlassian.net/browse/PRSDM-4454

### Checklist before merging

* [x] Has the changelog been updated?
* [x] Is the documentation updated for this change?

### CODE REVIEW

**Review by mpilo-khathwane — COMMENTED** [review]
🕺🏽

**Review by mpilo-khathwane — APPROVED** [review]

File: GitVersion.yml [hunk 1]
```diff
@@ -1,10 +1,18 @@
 mode: ContinuousDelivery
-next-version: 0.0.1
+next-version: 2.13.0
 branches:
+  feature:
+    mode: ContinuousDeployment
+    regex: ^feat[/-]
+    source-branches: ["develop", "master"]
```
Comment (Zalaras) [inline GitVersion.yml:7]: You have "main" in the .travis.yml file too?
Edit: also missing below x2

**Review by Zalaras — COMMENTED** [review]
Looks good to me. Just the one comment. You guys didn't want to use something like Semantic Release to handle this? Or is this mot quite the right use case

File: GitVersion.yml (hunk 1 above)
Comment (meyerhp) [reply GitVersion.yml:7]: Thanks for the review! Yeah, this was a copy and paste from enterprise-js and it slipped through. 

I'm not familiar with Semantic Release but I will check it out :)

**Review by Zalaras — APPROVED** [review]
LGTM

**Pull request merged**